haproxy/content.md

```diff
@@ -21,7 +21,10 @@ Note: Many configuration examples propose to put `daemon` into the `global` sect
 	FROM haproxy:1.5
 	COPY haproxy.cfg /usr/local/etc/haproxy/haproxy.cfg
 
-Build and run:
+## Test the configuration file	
+	docker run -it --rm  --name my-running-haproxy -v /path/to/haproxy.cfg:/usr/local/etc/haproxy/haproxy.cfg:ro haproxy:1.5 haproxy -c -f /usr/local/etc/haproxy/haproxy.cfg
```

Comment:
Yeah, sounds good!  Feel free to force push to the same branch you've
already got here (which will update the PR if you aren't familiar with the
GitHub flow), and give us a ping here when you're ready (since GitHub
doesn't notify when the PR is updated). :heart: